CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 0.10.0 (not yet released)
 
+- unified `waitForPorts` and `waitForPorts.timeoutSec` properties across providers
+- added possibility to wait for ports before a boot script is executed; it's configured by
+  `bootScript.waitForPorts` and `bootScript.waitForPorts.timeoutSec` properties
+- removed support for deprecated Azure Node properties `azure.userData`
```

Comment:
It was deprecated in `0.9.0` and should be removed in `0.10.0`. IMO it should be fine.